metacat-main/src/main/java/com/netflix/metacat/main/configs/ElasticSearchConfig.java

```diff
@@ -179,4 +180,40 @@ public ElasticSearchRefresh elasticSearchRefresh(
             registry
         );
     }
+
+    /**
+     * ElasticSearch refresh action.
+     *
```

Comment:
Think the refresh refers to the catalogTraversalAction, can we make the comment consistent?

Reply:
Will do.